src/views/List.jsx

```diff
@@ -7,6 +8,10 @@ export function List({ data }) {
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
+				{React.Children.toArray(
+					data.map(({ name }) => <ListItem name={name} />),
+				)}
+
 				{/**
 				 * TODO: write some JavaScript that renders the `data` array
 				 * using the `ListItem` component that's imported at the top
```

Comment:
Also please delete this TODO comment since you don't need it anymore :slightly_smiling_face:

Reply:
We will do just that. 
Thank you for highlighting it.